src/IIIFPresentation/API/Features/Manifest/Requests/GetManifest.cs

```diff
@@ -0,0 +1,49 @@
+﻿using API.Converters;
+using API.Features.Storage.Helpers;
+using API.Infrastructure.AWS;
+using MediatR;
+using Models.API.Manifest;
+using Repository;
+using Repository.Helpers;
+
+namespace API.Features.Manifest.Requests;
+
+public class GetManifest(
```

Comment:
Do the constructor values need to each be on separate line? It looks like it wouldn't go above the default JetBrains boundaries if they were moved into one line:
![image](https://github.com/user-attachments/assets/e8f7ac20-d403-418f-9465-d01283585a0c)

Reply:
I don't /really/ care either way, as long as it's automatic. We can use a ready-made settings or go through a process of generating appropriate settings files ourselves.

Also tbh I code on single 17" screen and I have mediocre eyesight :D